Sources/NIOEmbedded/Embedded.swift

```diff
@@ -15,13 +15,18 @@
 import Dispatch
 import _NIODataStructures
 import NIOCore
+import NIOConcurrencyHelpers
```

Comment:
I don't think `NIOEmbedded` depended on `NIOConcurrencyHelpers` prior to this, could you add the dependency explicitly to the target in `Package.swift`?

Reply:
Hate to be a pain but we now no longer need this import! Looks good otherwise.